app/models/Comment.php

```diff
@@ -0,0 +1,136 @@
+<?php
+namespace app\models;
+use app\core\Database;
+use app\interfaces\EntityInterface;
+use InvalidArgumentException;
+
+class Comment implements EntityInterface
+{
+    protected Database $db;
+
+    public function __construct() {
+        $this->db = new Database();
+    }
+    /** API Create a new Post
+     * @param array $data
+     * @return void
+     */
+    public function create(array $data): void
+    {
+        $this->db->query('INSERT INTO Comments (post_id, name, message, user_id, created_at, status) VALUES (:post_id,:name, :message, :userId, NOW(), "pending")');
+        $this->db->bind('post_id', $data['postId']);
+        $this->db->bind(':name', $data['name']);
+        $this->db->bind(':message',$data['message']);
+        $this->db->bind(':userId', $data['userId']);
+        $this->db->execute();
+    }
+
+    /** Model delete a post from the DB
+     * @param $id
+     * @return void
+     */
+    public function delete($id): void
```

Comment:
Given that the `delete` functions are almost identical in each Model, perhaps we could consider creating an `AbstractEntity` class to save on some of this repeated code.  

Also, each model has a `getModelById` function that could also benefit from this approach